src/vireshJJun20/ConvertCase.java

```diff
@@ -0,0 +1,67 @@
+/*
+ Assignement 29 : 17th July 2020
+
+WAP to convert case in all elements of array. 
+Input : 
+String[] array = {"AaVruti", "NiKhil", "AdiTi", "MAhesh", "ArchaNA", "jaGAdeesh"};
+
+Output:
+array = {"aAvRUTI", "nIkHIL","aDItI", "maHESH","aRCHAna","JAgaDEESH"};
+
+*/
+
+package vireshJJun20;
+
+public class ConvertCase {
+
+	static String tempString2 = "";
+
+	// Method accepts Array, converts each index to appropriate case and then saves
+	// in output array.
+	String[] StringConvertCase(String[] inputArray) {
+		String[] outputArray = inputArray;
+		for (int arrIndex = 0; arrIndex < inputArray.length; arrIndex++) {
+			String str = inputArray[arrIndex];
```

Comment:
we can remove the **str** variable here and just apply the **charAt** function on **inputArray[index].**

Reply:
Yes we can, however I had used it for more readability.